Shared/Samples/Create load report/CreateLoadReportView.Model.swift

```diff
@@ -0,0 +1,299 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+extension CreateLoadReportView {
+    /// The model used to store the geo model and other expensive objects
+    /// used in this view.
+    @MainActor
+    class Model: ObservableObject {
+        // MARK: Properties
+        /// The utility network for this sample.
+        private let utilityNetwork = UtilityNetwork(url: .utilityNetwork)
+        
+        /// The initial conditional expression.
+        private var initialExpression: UtilityTraceConditionalExpression!
+        
+        /// The trace parameters for creating load reports.
+        private var traceParameters: UtilityTraceParameters!
+        
+        /// The network attributes for the comparison.
+        private var phasesNetworkAttribute: UtilityNetworkAttribute!
+        
+        /// A list of possible phases populated from the network's attributes.
+        /// By default, they are not included in the load report.
+        @Published private(set) var excludedPhases = [CodedValue]()
+        
+        /// A list of phases that are included in the load report.
+        @Published private(set) var includedPhases = [CodedValue]()
+        
+        /// A list of possible phases populated from the network's attributes.
+        private var allPhases = [CodedValue]()
+        
+        /// The phase summaries in the load report.
+        @Published private(set) var summaries = [CodedValue: PhaseSummary]()
```

Comment:
```suggestion
       private var summaries = [CodedValue: PhaseSummary]()
```
This doesn't seem to be used in the UI.